recipes/sdl2/all/conanfile.py

```diff
@@ -269,67 +272,81 @@ def _build_cmake(self):
     def package(self):
         self.copy(pattern="COPYING.txt", dst="license", src=self._source_subfolder)
         cmake = self._configure_cmake()
-        cmake.install(build_dir=self._build_subfolder)
+        cmake.install()
+        tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), "sdl2-config")
         tools.rmdir(os.path.join(self.package_folder, "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "libdata"))
```

Comment:
I don't see this directory in my own Windows or Linux builds and I don't see it in macOS package?

Reply:
FreeBSD only: https://github.com/SDL-mirror/SDL/blob/76468465a94b59f0dde4791846bc2476be4f245f/CMakeLists.txt#L2190